lego_port.go

```diff
@@ -24,8 +24,13 @@ type LegoPort struct {
 	id int
 
 	// Cached values:
+	modes []string
```

Comment:
Confirm that this is not changed by a write to `set_device`?

Reply:
Confirmed. Writing to `set_device` will create a new `lego-sensor` device. It does not affect the port state in any way.